lib/typescript/httpclient/model/Tag.ts

```diff
@@ -0,0 +1,7 @@
+export type ColorTag = 'tag-blue' | 'tag-red' | 'tag-green' | 'tag-purple';
```

Comment:
I'd call the type `TagColor` I think, it seems more fitting? (naming is hard though and I may be getting this wrong)